src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718");
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public String getTask() {
+        return description;
+    }
+
+    public String toString() {
+        String str = "[";
+        if (isDone) {
+            str = str + getStatusIcon() + "] " + description;
+        } else {
+            str = str + getStatusIcon() + "] " + description;
```

Comment:
Perhaps you could consider consolidating the duplicate conditional fragments? Since getStatusIcon() also checks for isDone, maybe you would not need to check for isDone within toString().

Reply:
Thank you for the suggestion! I have done the necessary amendments!